src/System.Text.Json/src/System/Text/Json/Writer/JsonWriterHelper.Date.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+
+namespace System.Text.Json
+{
+    internal static partial class JsonWriterHelper
+    {
+        public static void TrimDateTime(Span<byte> buffer, out int bytesWritten)
+        {
+            Trim(buffer, out bytesWritten);
+        }
+
+        public static void TrimDateTimeOffset(Span<byte> buffer, out int bytesWritten)
```

Comment:
This split is unnecessary imo. I was thinking if the actual implementation needed to be split.
I am fine with the main workhorse method named as `TrimDateTimeOffset` and used for both if it truly is identical.